multicluster/config/samples/clusterset_init/member-clusterset-template.yml

```diff
@@ -22,7 +22,4 @@ spec:
     - clusterID: test-cluster-leader
       secret: leader-access-token
       server: https://<LEADER_CLUSTER_IP>:6443
-  members:
-    - clusterID: test-cluster-leader
-    - clusterID: test-cluster-member
```

Comment:
I think this line should be kept, there is a step to validate the value in ClusterClaim matched the ID in the member list. Or you think it's better to remove the step on member ClusterSet controller?

Reply:
I feel the current design is unnecessarily complex. In my mind, we just need a single ClusterSet CRD to define a ClusterSet, which should include leader information and the current cluster's ID. That is it. No ClusterClaims, useless fields and validation.

Back to this specific case, yes, I think we should remove "members" from the ClusterSet Spec, assuming we just use ClusterClaim to define the ClusterID. As we discussed earlier, probably we should consider to move members to Status in the leader cluster.

Reply:
Hi @jianjuns  here is the PR #4084 to remove the member ID validation, I didn't remove members from ClusterSet Spec considering it requires version upgrade. I suppose you'd like to merge this doc for v1.8?

Reply:
Sure. We can consider ClusterSet spec change later.

I feel spec change is not important. But in 1.8. users cannot re-apply ClusterSet yaml to avoid overriding the member list in the spec, if we do not move members to status?

Reply:
If the change in PR #4062 can be ready before v1.8, I think the status can be kept even user re-apply ClusterSet yaml since the status data are from MemberClusterAnnounce controller only.

Reply:
?? #4062 did not move member list from ClusterSet Spec to Status right? I meant if users re-apply ClusterSet yaml, the members added to ClusterSet Spec by MemberClusterAnnounce Reconciler will be overridden.

Reply:
Do we have a way to disallow member list change from other sources other than antrea-mc-controller, in the ClusterSet validation webhook? @luolanzone

Reply:
Yes, I think we can check the ServiceAccount to disallow regular users to change it. I will check this part.